### ISSUE TITLE
Mapq

### ISSUE THREAD

**Issue by nchernia** [brentp/bwa-meth#55]
I’ve noticed that running bwa meth results in a higher rate of MAPQ 0 reads than vanilla bwa (I did a test on non-bisulfite converted data). Is there any way to ameliorate this or is it just a natural consequence of needing to distinguish between methylated and nonmethylated loci?

**Comment by brentp** [comment]
[this](https://github.com/brentp/bwa-meth/blob/master/bwameth.py#L398) is the only place it's adjust mapq and it's setting to 1. mapq 0 means it maps to multiple places equally which is more likely to occur with a simplified reference.

**Comment by nchernia** [comment]
Yes, I was wondering on a more fundamental level; perhaps via the flags bwa-meth uses to call bwa. I asked Heng Li and he suggested asking here.

BTW, what's the motivation behind that heuristic?  Those are reads that bwa deems good enough to map, is there a reason more fundamental to bisulfite alignment that you would label them failing QC?

**Comment by brentp** [comment]
if you can show that it performs better without that, I'll remove it. I've actually found (for non-BS-Seq data) that alignments/ regions with high NM are often bad as well.

**Comment by nchernia** [comment]
I was thinking it should be an optional flag to turn it off (was eventually going to code it and make a pull request).  We expect chimeric reads in our experiment, but if one was not expecting chimeras, I could see it being useful.

**Comment by brentp** [comment]
still same criteria. if you can show that it is more accurate without, it can be removed. I'm hesitant to add more command-line flags.
